WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -94,11 +94,11 @@ struct TotalsView: View {
     private var backgroundColor: Color {
         switch posModel.paymentState {
         case .card(.cardPaymentSuccessful), .cash(.paymentSuccess):
-            .posSecondaryBackground
+            .posSurfaceContainerLowest
         case .card(.processingPayment):
-            colorScheme == .light ? Color(.wooCommercePurple(.shade70)) : Color(.wooCommercePurple(.shade10))
+            .posPrimary
```

Comment:
Thank you! With this change `processing...` view appears in the same color in both light and dark mode. Is this intentional? The only thing that changes then is the floating menu.

![Simulator Screenshot - iPad Pro 11-inch (M4) - 2025-02-07 at 10 14 35](https://github.com/user-attachments/assets/dc715827-6b45-4b72-b0ad-d7f15c1265cd)
![Simulator Screenshot - iPad Pro 11-inch (M4) - 2025-02-07 at 10 14 32](https://github.com/user-attachments/assets/ef988510-58b4-4f9c-b668-7e3e1ad02114)

Reply:
Yes, this is the color used in the design system: 1qcjzXitBHU7xPnpCOWnNM-fi-256_15446

The previous primary button (now called "filled" style in the design system) color is also the same in both light and dark mode now.